Test

BAProductions opened the pull request:
Test

Thecrazyskull committed:
Launcher3: double tap on home screen to turn off screen

Change-Id: I554c005d5e523aca0842c78a353686e86af1a7f2
Signed-off-by: Mesquita <mickaelmendes50@gmail.com>
Signed-off-by: Omkar Chandorkar <gotenksIN@aosip.dev>
Signed-off-by: Pranav <npv12@iitbbs.ac.in>

maxwen committed:
Launcher3: use common style for recents task menu divider

Change-Id: Ida14211a0d8decb897e857f44e071c51122eda9c
Signed-off-by: DennySPB <dennyspb@gmail.com>
Signed-off-by: Arghya Chanda <arghyac35@gmail.com>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

ezio84 committed:
New Recents: add kill action to app shortcuts popup

*DU Edits*
- Changed icon to match the icon style adopted by AOSP

Credit thanks to materialdesignicons site
http://materialdesignicons.com/icon/close-box-outline

Bootleggers edit: Update to Q changes

Change-Id: If137eb832aaafc4c0ee23767c34bcae745c78699
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

eldainosor committed:
Launcher3: Let's make the search bar on all apps round

Change-Id: Icd8eaa618720984f17319872e3fd12d87e934b86
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

Jertlok committed:
SystemShortcut: Add uninstall option

This allows to easily uninstall application on the popup
activity triggered by long pressing on the applications.

@neobuddy89: Add back support functions in Utilities.

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

amirzaidi committed:
Launcher3: Load backported notification icon from correct package

Conflicts:
	src/com/android/launcher3/notification/NotificationInfo.java

amirzaidi committed:
Launcher3: Keep spring adaptive icon hidden until drag starts

eldainosor committed:
Launcher3: new icon for kill app option

Square looks a bit old, why not use a circle?

Credits goes once again to http://materialdesignicons.com/

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

maxwen committed:
Launcher3: remove extra space for hotseat

Change-Id: I380f38c370e60453218db3d384014d04a1908348
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

Commit:
Launcher3: Consider extra header content as visible elements

Invoking extra header content such as floating header (predictions view)
causes the overview panel to show under recent task. The search bar doesn't
show in this case since it expects to scrim based on all apps content.

Since predictions appear in this area, set the search bar to show as well,
otherwise we get empty space above the predictions view.

Change-Id: I7eb68f4bc2efdc383f0ced5ff4a0ad13e9bed528
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

DennySPB committed:
Increase all apps top padding to avoid visual glitch in recents

Signed-off-by: DennySPB <dennyspb@gmail.com>
Change-Id: Ie7c995318b4fdf09c1de3220e12aeae2225cefbc
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

neobuddy89 committed:
Launcher3: Follow system color for dark style

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

neobuddy89 committed:
Launcher3: Use system fonts

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

amirzaidi committed:
Launcher3: Add AppInfo Bottom Sheet from Shade Launcher

Shade: Rename ShortcutFactory to ShadeShortcutFactory

Shade: Override AppInfoShortcut with BottomSheetShortcut

Shade: Import bottom sheet class from Pie

Shade: Fix ShadeShortcutFactory constructor

Shade: Add testing entries to App Info

Shade: Add real entries to App Info

Shade: Implement parsing for app info

Shade: Move all preference title strings to global strings file

Shade: Set source to "Unknown" if it is empty

Shade: Set icon pack to "None" by default

Shade: Update app info bottom sheet to Q widgets bottom sheet

Shade: Disable overscroll effect on bottom sheet preferences

Shade: Update Version app info preference icon

Shade: Reorder app info preference entries
Change-Id: I34fe46944561723d3a47db92b3814e1deee80b5b
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

maxwen committed:
Launcher3: add dark primary support

Change-Id: I2368b690571517dfb7f331832c9f52dcc8ac8531
Signed-off-by: RJ Trujillo <certifiedblyndguy@gmail.com>

maxwen committed:
Launcher3: fix dark primary folder bg color

Change-Id: I0355a5bdb21773783f965d9a5eae1ffee32d4206
Signed-off-by: RJ Trujillo <certifiedblyndguy@gmail.com>

Hikari-no-Tenshi committed:
Launcher3: Make double tap to sleep gesture optional

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

neobuddy89 committed:
Launcher3: Ensure no divider shown in preferences

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

gwolf2u committed:
disable promisse apps

neobuddy89 committed:
Launcher3: Fix font family for App Info sheet

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

bvlj committed:
Launcher3: update recent tasks header style

* Show the name of the task at the top
* Menu icon for split, info and pin

Change-Id: I2105fc269a565c712e0e664a5c9e68cb357025ac
Signed-off-by: Joey <joey@lineageos.org>

Commit:
Fix misleading "Search apps" pt translation

Fix the string "Apps de pesquisa" that roughly means "apps you can use to search", not the act searching an app as intended by the hint.

Change-Id: I65d91a9c0c5de49a76a55e10ec5f6364091da77a

SirRGB committed:
Revert "Launcher3: new icon for kill app option"

This reverts commit 6992f83255b1db381afd7bc4b2fff3ab104931ea.

SirRGB committed:
Revert "New Recents: add kill action to app shortcuts popup"

This reverts commit f47a7f8220f1054729a6c8c2b6b43d687dc5c231.

SirRGB committed:
Revert "Launcher3: use common style for recents task menu divider"

This reverts commit ab641c70924829c136d6b08b8f6a973fabc71f33.

BAProductions commented:
@green

BAProductions commented:
@BAProductions @birdupyee

BAProductions commented:
@greengeckowizard

birdup000 commented:
> @greengeckowizard
@BAProductions 
My TG account got banned I'll communicate on the Xda forums for now until it's been resolved.

BAProductions commented:
Ok

SirRGB committed:
[DNM] disable top shadow on statusbar

* looks horrible on oled, but could not find any commit making it optional for A10 trebuchet.

pinyaoting committed:
Fix permission issue in legacy shortcut

When building legacy shortcut, Launcher calls
PackageManager#resolveActivity to retrieve necessary permission to
launch the intent.

However, when the source app wraps an arbitrary intent within
Intent#createChooser, the existing logic will fail because launching
Chooser doesn't require additional permission.

This CL fixes the security vulnerability by performing the permission
check against the intent that is wrapped within.

Bug: 270152142
Test: manual
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:c53818a16b4322a823497726ac7e7a44501b4442)
Merged-In: If35344c08975e35085c7c2b9b814a3c457a144b0
Change-Id: If35344c08975e35085c7c2b9b814a3c457a144b0

SirRGB committed:
use system fonts

pinyaoting committed:
Fix permission bypass in legacy shortcut

Intent created for Chooser should not be allowed in legacy shortcuts
since it doesn't make sense for user to tap on a shortcut in homescreen
to share, the expected share flow started from ShareSheet.

Bug: 295334906, 295045199
Test: manual
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:b7b192bd7f24a2aa7d6881ee949657c9760c0305)
Merged-In: I8d0cbccdc31bd4cb927830e5ecf841147400fdfa
Change-Id: I8d0cbccdc31bd4cb927830e5ecf841147400fdfa
